.config/wt/README.md

```
@@ -0,0 +1,164 @@
+
+### Cleanup merged branches
+
+```bash
+wt step prune
+```
+
+## Keybinds for Option Left/right to switch tabs
+
+In `~/.config/zellij/config.kdl` add:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)